client/sections.js

```diff
@@ -222,6 +222,14 @@ if ( config.isEnabled( 'reader' ) ) {
 	} );
 
 	sections.push( {
+		name: 'reader-post-recomendations',
+		paths: [ '/recommendations' ],
```

Comment:
i think this needs to be more specific, and possibly above the other recs section. What gets loaded when you try to load recs?